server/webhook.go

```diff
@@ -425,6 +380,186 @@ func (s *Server) createTenantSchemaUpdateIssue(mi *db.MigrationInfo, vcsPushEven
 	return string(createContext), nil
 }
 
+// createIssueFromPushEvent attempts to create a new issue for the given file of
+// the push event. It returns "created=true" when a new issue has been created,
+// along with the creation message to be presented in the UI. An *echo.HTTPError
+// is returned in case of the error during the process.
+func (s *Server) createIssueFromPushEvent(ctx context.Context, repo *api.Repository, pushEvent vcs.PushEvent, file, webhookEndpointID string) (message string, created bool, _ *echo.HTTPError) {
```

Comment:
nit: avoid using named result parameters.

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
I believe this case fits "on the other hand":

<img width="1052" alt="CleanShot 2022-07-24 at 21 05 00@2x" src="https://user-images.githubusercontent.com/2946214/180648315-b4bdd769-d25a-4aba-94b7-004ff31227a2.png">

Where named return values provide clarity.

Reply:
The three parameters have different types. Also, the third parameter could just be type "error".

Reply:
> [...], or if the meaning of a result isn't clear from context, adding names may be useful in some contexts.

This reasoning doesn't need return values to have the same type.

>Also, the third parameter could just be type "error".

Will address this.

Reply:
"the meaning of a result isn't clear from context": this is subjective so I'm not gonna argue on this. LGTM.